flake.nix

```diff
@@ -8,10 +8,12 @@
     flake = false;
   };
   inputs.devenv = {
-    url = "github:vlaci/devenv/python-wrapper";
+    url = "github:cachix/devenv/v1.4";
```

Comment:
Just a question: Why pin it for a specific version? Isn't flake.lock used for that?

Reply:
I was afraid it being too bleeding edge, but as renovate can update flake.lock by input, it may be a moot point.